app/helpers/application_helper/button/smart_state_scan.rb

```diff
@@ -10,6 +8,14 @@ def check_smart_roles
     end
   end
 
+  def calculate_properties
+    super
+    if disabled?
```

Comment:
@yaacov @himdel Checked this with @vecerek and this change doesn't seem right.
- I don't think that setting `self[:onwhen]` on `nil` would help you here, as it's overwritten by JavaScript.
- There is no need to set `self[:title]` again (you are calling it by `super`) as the `disabled?` method is already called in `basic.rb` and sets the correct error message
- Also carefull with changes here, as the class is being inherited by other button classes